site/public/mjs/sql-toolbox.js

```diff
@@ -24,7 +24,7 @@ async function runSqlQuery() {
         table.innerHTML = '';
 
         if (json.status !== 'success') {
```

Comment:
It's weird how codecov reporting a missing branch here.

Reply:
Looks at the code coverage report locally it shows both branches coverage so maybe its a codecov specific issue